config.example.yaml

```
@@ -0,0 +1,139 @@
+  - https://subgraph-url2.com
+
+# Arb contract address, required
+arbAddress: "0x1234...5678"
+
+# Generic arb contract address, required
+genericArbAddress: "0x1234...5678"
+
+# Dispair contract address, required
+dispair: "0x1234...5678"
+
+# list of liquidity providers 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)